lib/components/narrative/save-trip-button.js

```diff
@@ -16,31 +18,32 @@ const SaveTripButton = ({
   loggedInUser,
   persistence
 }) => {
+  const intl = useIntl()
   // We are dealing with the following states:
   // 1. Persistence disabled => just return null
   // 2. User is not logged in => render something like: "Please sign in to save trip".
   // 3. itin cannot be monitored => disable the button with prompt and tooltip.
 
   let buttonDisabled
-  let buttonText
-  let tooltipText
-  let icon
+  let buttonTextId
+  let tooltipTextId
+  let iconType
 
   if (!persistence || !persistence.enabled) {
     return null
   } else if (!loggedInUser) {
     buttonDisabled = true
-    buttonText = 'Sign in to save trip'
-    icon = 'fa fa-lock'
-    tooltipText = 'Please sign in to save trip.'
+    buttonTextId = 'components.SaveTripButton.signInText'
+    iconType = 'lock'
+    tooltipTextId = 'components.SaveTripButton.signInTooltip'
   } else if (!itineraryCanBeMonitored(itinerary)) {
     buttonDisabled = true
-    buttonText = 'Cannot save'
-    icon = 'fa fa-ban'
-    tooltipText = 'Only itineraries that include transit and no rentals or ride hailing can be monitored.'
+    buttonTextId = 'components.SaveTripButton.cantSaveText'
+    iconType = 'ban'
+    tooltipTextId = 'components.SaveTripButton.cantSaveTooltip'
```

Comment:
Similar comment as the one above

Reply:
Updated in https://github.com/opentripplanner/otp-react-redux/pull/430/commits/b28ccbdecf85d6e381b8a03c27b0e2910bda849f.